sdk/spring/spring-cloud-azure-core/src/main/java/com/azure/spring/core/resource/AbstractAzureStorageProtocolResolver.java

```diff
@@ -32,8 +32,20 @@
      */
     protected final AntPathMatcher matcher = new AntPathMatcher();
 
+    /**
+     * The storageType of ProtocolResolver
+     * For AzureStorageBlobProtocolResolver, getStorageType() will return StorageType.BLOB;
+     * For AzureStorageFileProtocolResolver, getStorageType() will return StorageType.FILE;
```

Comment:
How about moving these descriptions to the implementation classes, as an abstract class, we perhaps won't know how many implementations it will have.